ResolvedTitle.md

```diff
@@ -2,17 +2,18 @@
 
 **URL:** https://www.api.careerbuilder.com/core/classifier/resolvedtitle
 
-This endpoint supports the HTTP/POST method only.
+This endpoint supports the both HTTP/GET and HTTP/POST methods.
 
 **Parameters**
++(This API is in transition to match our data science API input/output standard. To provide peaceful transition, we support versions 0.9 and 1.0. Version is passed in the Accept header as the parameter version (Ex. Accept:application/json;version=1.0). 0.9 is the default version if user does not specify explicitly and all our existing users use this version by default. We highly encourage new users to pass version 1.0 in the Accept header. In version 0.9, all input parameters and output json fields are in camel case, while in version 1.0, input parameters and output json fields are in lowercase with underscores.)
```

Comment:
Good explanation of this. I would just change the first sentence to say "to match CareerBuilder API standards" (the standards aren't limited to our group). Also maybe add a sentence at the end stating that 0.9 will be retired in a few months.